pkg/kubernetes/scripts/projects.js

```diff
@@ -937,14 +966,20 @@
                     displayName: fields.display.trim(),
                     description: fields.description.trim()
                 };
-
-                return methods.check(request, { "metadata.name": "#project-new-name" })
-                    .then(function() {
-                        return methods.create(request)
-                            .then(function() {
-                                return projectData.shareImages(name, fields.access === "shared");
-                            });
-                    });
+                var chain = $q.when();
+                chain = chain.then(function() {
+                    return methods.check(request, { "metadata.name": "#project-new-name" });
+                });
+                chain = chain.then(function() { 
+                    return methods.create(request);
+                });
+                chain = chain.then(function() { 
+                    return projectData.shareImages(name, fields.access === "shared");
+                });
+                chain = chain.then(function() {
+                    return projectData.makeRegistryAdmin(name, dialogData.currUser);
```

Comment:
@stefwalter @aweiteka 
for time being we need this to make registry-admin useful in our UI